bundles/org.openhab.binding.enocean/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -449,4 +450,85 @@
 		<category>thermostat</category>
 	</channel-type>
 
+	<channel-type id="smokeDetection">
+		<item-type>Switch</item-type>
+		<label>Smoke Detected</label>
+		<description>Smoke detection sensor state.</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="sensorFault">
+		<item-type>Switch</item-type>
+		<label>Sensor Fault Mode</label>
+		<description>Sensor activated fault mode.</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="maintenanceStatus">
+		<item-type>Switch</item-type>
+		<label>Maintenance Neccessary</label>
+		<description>Sensor maintenance status.</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="saHumidityRange">
+		<item-type>Switch</item-type>
+		<label>Out of Humidity Range</label>
+		<description>Sensor analysis status: humidity out of range</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="saTemperatureRange">
+		<item-type>Switch</item-type>
+		<label>Out of Temperature Range</label>
+		<description>Sensor analysis status: temperature out of range</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="timeSinceLastMaintenance">
+		<item-type>Number:Time</item-type>
+		<label>Time Since Last Maintenance</label>
+		<description>Time since last maintenance was done.</description>
+		<state readOnly="true" />
```

Comment:
Do we need a pattern for the `QuantityType` items?

```suggestion
		<state readOnly="true" pattern="%.1f %unit%" />
```

Reply:
Missed to add it, as it was a simple number channel in first place. Thanks for the hint 👍